### ISSUE TITLE
Add System Automation Scripts

### ISSUE THREAD

**Issue by powerexploit** [powerexploit/Awesome-Python-Scripts#94]
# This is hard level issue.

- **Scripts allow you to automate tasks not only for fun fact but also some important tasks.**
- **You can automate anything specially command line of Windows,Linux,Ubuntu or Mac.**
- **So try to add some advance and fun based system automation scripts in System-Automation-Scripts folder.**

# Comment on this issue and start contributing.

**Comment by kritikaparmar-programmer** [comment]
Hii @ankitdobhal , Can I add script for automating google search? I want to work on this issue!

**Comment by powerexploit** [comment]
> Hii @ankitdobhal , Can I add script for automating google search? I want to work on this issue!

Yes sure but contest will start from 16th august start from then.

**Comment by kritikaparmar-programmer** [comment]
Ok 👍

**Comment by Arsenic-ATG** [comment]
@ankitdobhal if our script comes under both automation and web-scraping then which one should be preferred over other ?
means should we post it under automation or web-scraping ?

**Comment by powerexploit** [comment]
> @ankitdobhal if our script comes under both automation and web-scraping then which one should be preferred over other ?
> means should we post it under automation or web-scraping ?

Yes web scraping script will be under the Web-Scraping folder with the respective #92 issue.

**Comment by sharur7** [comment]
Hello, @ankitdobhal I want to work on the script for automating sending emails. I want to work on this issue.

**Comment by PatelKeviin** [comment]
Hi @ankitdobhal, I'd also like to start working on this issue as I am proficient in automation tasks. Thanks!

**Comment by avinashkranjan** [comment]
Hi  I'd also like to start working on this issue..

**Comment by peeky1811** [comment]
Hi @ankitdobhal. Can I make a Flipkart Price Tracker using Python and BeautifulSoup to send an email by taking the URL of the product page, Name of the product and Email-id to be contacted on?
Gopika Murali
Participant at Contributor's Hack '20, HakinCodes

**Comment by subhayu99** [comment]
Hi @ankitdobhal 

I would like to add a **QR Code Generator** to the repository which converts URLs to QR using Python

Subhayu Kumar Bala
Participant at CH20

**Comment by Namyalg** [comment]
Hi @ankitdobhal , I would like to work on automating the process of sending whatsapp messages.

**Comment by Namyalg** [comment]
I am a participant at CH20

**Comment by AdityaJ7** [comment]
I would like to work on this issue

CH20 participant

**Comment by Success0452** [comment]
I will work on this issue

**Comment by Namyalg** [comment]
I would like to write a script for tracking the price of an Amazon product and sending an automated email.
Namya LG
CH20 Participant

**Comment by piyushP7pravin** [comment]
Hey, I want to work on Instagram Automation.

**Comment by gupta-niharika** [comment]
Hi i would like to contribute towards this issue. I could work towards mass mailing.

**Comment by DeepNinja07x** [comment]
Hi @ankitdobhal, @Neilblaze
I want to contribute to this issue. I would like to add an automated text to voice converter program.

Deepraj Rakshit
Participant at CH20

**Comment by Neilblaze** [comment]
@piyushP7pravin & @gupta-niharika Sure! Go for it :+1:

**Comment by sharur7** [comment]
@ankitdobhal @Neilblaze I want to work on this issue and contribute Twitter Automation code.
Participant at Contributor's Hack '20, HakinCodes
#168

**Comment by sharur7** [comment]
#168

**Comment by souryankit** [comment]
@ankitdobhal sir i would like to submit my pull request #141 in this issue #94 .. plz chek my pull request and merge it.

**Comment by Namyalg** [comment]
Hi @ankitdobhal , please review issue #116.

**Comment by Neilblaze** [comment]
A message to every participants, mentors and PA's are trying their best to review and help your PR get merged. Have faith upon us, do spare us some time to review individual PR's.

Earnest co-operation is highly solicited!

**Comment by Namyalg** [comment]
I would like to work on a script to convert text to speech

**Comment by abhishek-mishra17** [comment]
hi @ankitdobhal  I have added a python script to get current location , please review my pull request #229 , I have added the output in the readme file as you had asked me , please fell free to give any suggestions.

**Comment by Namyalg** [comment]
Hi @ankitdobhal , I would like to know if I have to make any changes to issue #214.

**Comment by yashaswibiyahut** [comment]
I would like to automate the whole commit process using git. please assign me @ankitdobhal

**Comment by Neilblaze** [comment]
@Namyalg reviewed!

@yashaswibiyahut Sure! Create a PR & mention me or Ankit over there!

**Comment by Yuvraj-kadale** [comment]
Hello @ankitdobhal  I would like to automate google meet joining at a scheduled time. (keeping the camera and microphone off)
kindly, assign it to me!

**Comment by Neilblaze** [comment]
@Yuvraj-kadale Go for it! :+1:

**Comment by kaustubhgupta** [comment]
@ankitdobhal Made PR #263  @Neilblaze review  and merge !

**Comment by simarpreetsingh-019** [comment]
@Namyalg  @ankitdobhal I would like to make a Typing speed tester using python.  Please assign me this issue.

**Comment by simarpreetsingh-019** [comment]
I want to make a story teller using Python. #271 . Please assign me this.

**Comment by Neilblaze** [comment]
@simarpreetsingh-019 Sure! go for it. :+1:

**Comment by simarpreetsingh-019** [comment]
@Neilblaze could you please add label to my issue ? Would be great help.

**Comment by harshareddy794** [comment]
@ankitdobhal  I want to work on this issue

**Comment by Yuvraj-kadale** [comment]
@Neilblaze #284  changes are done as said...kindly review!!

**Comment by harsh7x18** [comment]
I would like to work on this issue. I will write python script to automate movies and TV series search.

**Comment by pyGitHub7** [comment]
Added the script, Kindly review!

**Comment by pyGitHub7** [comment]
> I would like to work on this issue. I will write python script to automate movies and TV series search.

Added the script, kindly review!

**Comment by harsh7x18** [comment]
> I would like to work on this issue. I will write python script to automate movies and TV series search.

Automated movies and TV series search #324. Added the python script, kindly review!
